packages/core/src/application.ts

```diff
@@ -50,6 +52,8 @@ export class Application extends Context {
       this._handleHttpRequest(req, res);
 
     this.bind('logError').to(this._logError.bind(this));
+    this.bind('send').to(writeResultToResponse);
```

Comment:
Makes sense, I always had a bit of a weird feeling about using "global" non-descriptive names like "send"/"reject".

I like `sequence.actions.send` 👍